x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/infrastructure/infrastructure_attributes.spec.ts

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Did we move this file from `apm_api_integration` folder to `deployment_agnostic` or duplicate it? Ideally we should move and make the adjustments without duplicating the tests.

Reply:
yeah, sorry, I forgot to delete, I will move it instead